projects/kubernetes-sigs/cluster-api-provider-cloudstack/CHECKSUMS

```diff
@@ -1,2 +1,2 @@
-1aa7731c14010ebe2b18b0ad64d4ad80ef92504c4474c617234d17e2c69bfe4d  _output/bin/cluster-api-provider-cloudstack/linux-amd64/manager
-37ab42d9d364e50bec7f6401a2178669ecb0cb135666a8dc9ff4b5673209068f  _output/bin/cluster-api-provider-cloudstack/linux-arm64/manager
+fa9e42d3c05687ca6594d4d72b4d292a43fa935c577c452a3f170b7fb4df20f4  _output/bin/cluster-api-provider-cloudstack/linux-amd64/manager
+d39fde8fd54982dd758677f767d30b60951866f6905d47a8cfa0f7e654e8f39a  _output/bin/cluster-api-provider-cloudstack/linux-arm64/manager 
```

Comment:
I imagine this space  is going to cause problems. Also the lack of newline at the end. Not sure how the comparison is done though